opentelemetry-api/java-agent/TestAzureCore/pom.xml

```diff
@@ -0,0 +1,121 @@
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>2.7.18</version>
+    <relativePath/> <!-- lookup parent from repository -->
+  </parent>
+
+  <artifactId>TestAzureCore</artifactId>
+
+  <properties>
+    <java.version>1.8</java.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core</artifactId>
+      <version>1.49.1</version>
+    </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-http-netty</artifactId>
+      <version>1.15.1</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpclient</artifactId>
+      <version>4.5.13</version>
+    </dependency>
+    <dependency>
+      <groupId>commons-logging</groupId>
+      <artifactId>commons-logging</artifactId>
+      <version>1.1.1</version>
+    </dependency>
+<!--    <dependency>-->
+<!--      <groupId>com.microsoft.azure</groupId>-->
+<!--      <artifactId>applicationinsights-agent</artifactId>-->
+<!--      <version>3.5.4-SNAPSHOT</version>-->
+<!--      <scope>system</scope>-->
+<!--      <systemPath>${project.basedir}/src/main/resources/applicationinsights-agent-3.5.4-SNAPSHOT.jar</systemPath>-->
+<!--    </dependency>-->
+  </dependencies>
+<!--  <build>-->
+<!--    <plugins>-->
+<!--      <plugin>-->
+<!--        <groupId>org.springframework.boot</groupId>-->
+<!--        <artifactId>spring-boot-maven-plugin</artifactId>-->
+<!--        <configuration>-->
+<!--          <finalName>app</finalName>-->
+<!--        </configuration>-->
+<!--      </plugin>-->
+<!--      <plugin>-->
+<!--        <groupId>org.apache.maven.plugins</groupId>-->
+<!--        <artifactId>maven-resources-plugin</artifactId>-->
+<!--        <version>3.3.1</version>-->
+<!--        <executions>-->
+<!--          <execution>-->
+<!--            <id>copy-resources</id>-->
+<!--            <goals>-->
+<!--              <goal>copy-resources</goal>-->
+<!--            </goals>-->
+<!--          </execution>-->
+<!--        </executions>-->
+<!--        <configuration>-->
+<!--          <outputDirectory>${project.build.directory}/agent/</outputDirectory>-->
+<!--          <resources>-->
+<!--            <resource>-->
+<!--              <directory>${project.basedir}/src/main/resources</directory>-->
+<!--              <includes>-->
+<!--                <include>applicationinsights-agent-3.5.4-SNAPSHOT.jar</include>-->
+<!--              </includes>-->
+<!--            </resource>-->
+<!--          </resources>-->
+<!--        </configuration>-->
+<!--      </plugin>-->
+<!--    </plugins>-->
+<!--  </build>-->
+
```

Comment:
did you want to keep this commented out code?

Reply:
yes, that is how we use a local copy of jar. i can reference it next time whenever needed.

Reply:
it's intentional.